packages.config

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Win32.Registry" version="4.5.0" targetFramework="net461" />
```

Comment:
I suggest to remove this file and packages folder due to new csproj format is used. We could use [PackageReference](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files) in csproj instead.